include/fastdds/rtps/writer/ReaderLocator.h

```diff
@@ -179,7 +181,8 @@ class ReaderLocator : public RTPSMessageSenderInterface
 
 private:
 
-    RTPSParticipantImpl* owner_;
+    RTPSParticipantImpl* participant_owner_;
+    RTPSWriter* owner_;
```

Comment:
Just a matter of taste:

```suggestion
    RTPSWriter* owner_;
    RTPSParticipantImpl* participant_owner_;
```